packages/web-components/src/components/va-radio/va-radio.tsx

```diff
@@ -220,24 +220,24 @@ export class VaRadio {
         'usa-legend': true,
         'usa-label--error': error
       });
-      return ( 
+      return (
         <Host
           aria-invalid={error ? 'true' : 'false'}
           aria-label={ariaLabel}
         >
           <fieldset class="usa-form usa-fieldset">
-            <legend class={legendClass}>
+            <legend class={legendClass} part="legend">
               {label}
-              {required && <span class="usa-label--required"> {i18next.t('required')}</span>}
+              {required && <span class="usa-label--required" part="required"> {i18next.t('required')}</span>}
```

Comment:
Is there a specific reason that you need to override the required text style?

Reply:
We're enlarging the `legend` text to match an `H3`, but the required label needs to be "normal" text sized

<img width="540" alt="legend is H3-sized and the required label is not bold and smaller in size" src="https://user-images.githubusercontent.com/136959/207378697-722bb90d-8c96-470f-9694-3cc588afcfe3.png">

Reply:
Just to note, maybe the stylesheets aren't in place yet? But I added a `uswds` property to the `va-radio` and everything got really tiny

<img width="673" alt="legend and option labels are tiny" src="https://user-images.githubusercontent.com/136959/207380149-ce562135-8d94-431e-9fab-404e2c673f8d.png">
<img width="369" alt="font size set to 1.06 rem" src="https://user-images.githubusercontent.com/136959/207380144-6b29685d-8a86-4131-bf0a-3e934de22553.png">

Reply:
We have run into an issue where Formation sets the root font size to 10px but USWDS v3 is expecting 16px so the variation wasn't rendering the correct font size. We solved for this in Storybook with this [toggle approach](https://github.com/department-of-veterans-affairs/component-library/pull/577/files) but I don't think it has been solved yet for vets-website. Thanks for bringing it up though because I think we missed creating a task for it (which [I've done here](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1402)).